### DISCUSSION TITLE
Is `renderErrorToHTML` deprecated/unused?

### DISCUSSION THREAD

**Discussion by therealgilles** [vercel/next.js#49475] [Help]
### Summary

I have been doing research on how to make sure all server errors are correctly captured and reported. I stumbled upon older issues adding an error handler to `renderErrorToHTML`. Looking at the current nextjs code, I don't see this function called anywhere but it is somehow still in the code.

Is `renderErrorToHTML` deprecated/unused? if so, why is it still in the code?

**Comment by jackleslie** [comment]
It can be used by custom servers, however it recently stopped working for this use case

Relevant [issue](https://github.com/vercel/next.js/issues/56936)
